Add_User.py

```diff
@@ -0,0 +1,13 @@
+def Add_User():
+    file = open("accountfiles.txt","a")
+    #Calculates ID using the length of the file
+    with open(r"accountfiles.txt", 'r') as fp:
+        count = 1
+        for line in enumerate(fp):
+            count += 1
+        userID = count
+    account = Account(input("Please enter your full name:"),input("Please enter your address: "),input("Please enter the amount of money you would like to deposit: "),str(userID))
```

Comment:
Maybe it is better to just store one by one to make a more visual solution:
name = input("Please enter your full name:")
....
account = Account(name, ..., ... , ...)

Reply:
Will do